powerauth-push-server/src/main/java/com/wultra/push/controller/rest/AdministrationController.java

```diff
@@ -74,14 +74,14 @@ public ObjectResponse<GetApplicationListResponse> listApplications() {
     @Operation(summary = "List unconfigured applications", description = "List unconfigured application in Push Server")
     public ObjectResponse<GetApplicationListResponse> listUnconfiguredApplications() throws PushServerException {
         try {
-            logger.debug("action: listUnconfiguredApplications, state: initiated");
+            logger.info("action: listUnconfiguredApplications, state: initiated");
             final List<PushServerApplication> applications = administrationService.findUnconfiguredApplications();
             final GetApplicationListResponse response = new GetApplicationListResponse();
             response.setApplicationList(applications);
-            logger.debug("action: listUnconfiguredApplications, state: succeeded");
+            logger.info("action: listUnconfiguredApplications, state: succeeded, size: {}", applications.size());
             return new ObjectResponse<>(response);
         } catch (PowerAuthClientException ex) {
-            logger.warn(ex.getMessage(), ex);
+            logger.error("action: listUnconfiguredApplications, state: failed, error: {}", ex.getMessage());
```

Comment:
I do not like this approach. The root exception is swallowed. The only message is logged, which is not enough. We should pass it at least as the cause of the exception that is thrown later on.

Reply:
Double-check the whole pull request for the same issue.